types/types.go

```diff
@@ -38,6 +38,8 @@ type NetConf struct {
 	RawDelegates []map[string]interface{} `json:"delegates"`
 	Delegates    []*DelegateNetConf       `json:"-"`
 	Kubeconfig   string                   `json:"kubeconfig"`
+	Logging      string                   `json:"logging"`
```

Comment:
Can `json:"logging` be `json:"log_file`

Reply:
Many plugin does not use '_' like [bridge](https://github.com/containernetworking/plugins/tree/master/plugins/main/bridge). How about to change logFile?

Reply:
I find that I name 'logging_level' just below...
I also fix it to 'loggingLevel'...